docs/postgres.md

```diff
@@ -88,6 +88,14 @@ dropdb opencollective_test
 dropuser opencollective
 ```
 
+## Migrations
+
+When creating migrations and interacting with the database please follow the guidelines below; 
+
+https://docs.opencollective.com/help/contributing/development/postgres-database
+
+**Note:** We use [Sequelize](https://sequelize.org/). Therefore, it's important to create the migrations with `npm run db:migration:create -- --name <migration-script-name>` so that it aligns with the default Sequelize file naming conventions. 
+
```

Comment:
Simplify and be straight to the point:

> To create a migration, always use ...

> For more information on the following page: https://docs.opencollective.com/help/contributing/development/postgres-database

Perhaps the documentation page should actually be deleted and all the content moved here.

Reply:
Done. I've simplified the language a bit and also moved all the migration sections from the doc here. I'll opened a draft in the docs at, https://app.gitbook.com/@opencollective/s/help/~/diff/drafts/-MiRTe33V4DdSc2GXqp5/@drafts to remove the [migrations page](https://docs.opencollective.com/help/contributing/development/postgres-database). After this is merged I'll merge that.